ibm_mq/datadog_checks/ibm_mq/collectors/channel_metric_collector.py

```diff
@@ -72,6 +73,9 @@ def get_pcf_channel_metrics(self, queue_manager):
                 self._submit_metrics_from_properties(
                     channel_info, channel_name, metrics.channel_metrics(), channel_tags
                 )
+        finally:
+            if pcf is not None:
+                pcf.disconnect()
```

Comment:
Is there a chance of this throwing an exception? (Should it be in its own try-except block?)

Reply:
[`disconnect()` ](https://github.com/dsuch/pymqi/blob/v1.12.0/code/pymqi/__init__.py#L2894-L2904)has a try-except block already, so I think it should be safe?